public/stylesheets/app/_header.scss

```diff
@@ -9,3 +9,7 @@
   cursor: pointer;
   text-decoration: underline !important;
 }
+
+.heading-count-order{
```

Comment:
You should prefix `pac` to additionally required classnames. So this should be `pac-heading-count-order` though I'm not sure you need this additional class as you could probably use one of the GOV.UK caption classes:

See: https://design-system.service.gov.uk/styles/typography/